src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml

```diff
@@ -90,7 +90,7 @@ fos_rest:
     format_listener:
         rules:
             - { path: '^/api', priorities: ['json', 'xml'], fallback_format: json, prefer_extension: true }
-            - { path: '^/', stop: true }
```

Comment:
Hi, could you please take a look at this issue https://github.com/Sylius/Sylius/issues/2421 which was the consequence of the `stop: true` removal